mission_yaml/SEA55_M35.yml

```diff
@@ -0,0 +1,456 @@
+metadata:
+  acknowledgement: This study used data collected and made freely available by Voice of the Ocean Foundation
+                   (https://www.voiceoftheocean.org) accessed from https://erddap.voiceoftheocean.org
+  institution: Voice of the Ocean Foundation
+  license: Creative Commons Attribution 4.0 (https://creativecommons.org/licenses/by/4.0/)
+    This study used data collected and made freely available by Voice of the Ocean Foundation
+    (https://www.voiceoftheocean.org) accessed from https://erddap.voiceoftheocean.org
+  format_version: IOOS_Glider_NetCDF_v2.0.nc
+  glider_model: SeaExplorer
+  glider_instrument_name: seaexplorer
+  keywords: "AUVS, Autonomous Underwater Vehicles, Oceans, Ocean Pressure,
+             Water Pressure, Ocean Temperature, Water Temperature,
+             Salinity/Density, Conductivity, Density, Salinity"
+  keywords_vocabulary: GCMD Science Keywords
+  metadata_link: http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html
+  Metadata_Conventions: CF-1.6, Unidata Dataset Discovery v1.0
+  naming_authority: Voice of the Ocean Foundation
+  platform:    "SeaExplorer Glider"
+  processing_level: "Data provided as is with no expressed or implied
+                     assurance of quality assurance or quality control. L0"
+  publisher_email: callum.rollo@voiceoftheocean.org
+  publisher_name:  Callum Rollo
+  publisher_url:   https://voiceoftheocean.org
+  references:     "created with pyglider https://github.com/c-proof/pyglider"
+  source:     Observational data from a profiling glider.
+  standard_name_vocabulary: CF STandard Name Table v49
+  transmission_system: IRIDIUM
+  glider_name: Kaprifol
+  glider_serial: "55"
+  wmo_id: "6801674"
+  comment: 'deployment and recovery in Kattegat'
+  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin
+  contributor_role: Data Processor, PI, Head of Operations, Glider Technician, Glider Technician
+  creator_email: callum.rollo@voiceoftheocean.org
+  creator_name: Callum Rollo
+  creator_url: https://callumrollo.com/
+  deployment_id: "35"
+  deployment_name: SAMBA
+  # date format 'yyyy-mm-dd'
+  deployment_start: "2022-03-24"
+  deployment_end: "2022-04-23"
+  project: SAMBA
+  project_url: https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/
+  summary: ' Part of SAMBA continuous monitoring'
+  # Examples: Skagerrak, Kattegatt, Baltic, Gulf of Bothnia, Gulf of Finland, Gulf of Riga
+  sea_name: Kattegat
```

Comment:
I think that we should avoid using Kattegat as the sea_name as we are actually not in Kattegat. It is fine for us to use that internally, but since this is metadata for the users it should probably be changed.